config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,28 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-build-and-push-k8s-at-golang-tip
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: build-and-push-k8s-at-golang-tip
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20200302-ca9526d
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/perf-tests=master
+        - --timeout=60
+        - --scenario=execute
+        - --
+        - make
+        - --
+        - --directory=/go/src/k8s.io/perf-tests/golang
+        - K8S_COMMIT=885e1e929a2b6cad671497cf1a12c6c66664614f
```

Comment:
Let's set this to k8s release-1.18 head as of today and add a comment saying that

Reply:
Done.